### PR TITLE
Add 'copyToClipboard' function in common.js + a few fixes

### PR DESCRIPTION
**Pull request by Micdu70**

### CODE REVIEW

File: plugins/bulk_magnet/init.js [hunk 1]
```diff
@@ -50,13 +38,13 @@ if(plugin.canChangeMenu())
...
-				break;	
+				break;
 			}
 		}
 		return(ret);
 	}
 
-	plugin.copyProperty = function(type)
+	theWebUI.copyProperty = function(type)
 	{
```
Comment (Novik) [inline plugins/bulk_magnet/init.js:48]: Do you really have plan to use 'theWebUI.copyProperty' somewhere else?
If no, then do not extend global interface 'theWebUI', use local 'plugin' interface instead, please.
Pollution of global interfaces with a non-common functions is not a good thing.

Comment (Micdu70) [reply plugins/bulk_magnet/init.js:48]: I really would like to keep using the 'plugin' interface but it doesn't work:
=> "plugin.copyProperty()":
----- Get `plugin is not defined` error when using double quotes in 'theContextMenu.add' call.
----- Without double quotes, all 3 'copyProperty' functions are called when the context menu is opened in ruTorrent.
=> "plugin.copyProperty" without "()" at the end:
----- Get `plugin is not defined` error when using double quotes in 'theContextMenu.add' call.
----- Without double quotes, copyProperty function is never called (and that's why I created this PR).

Comment (Novik) [reply plugins/bulk_magnet/init.js:48]: What about previous implementation, is it work?

Comment (Micdu70) [reply plugins/bulk_magnet/init.js:48]: Previous implementation doesn't seem to work (at least not with my personal ruTorrent installation). I don't know the last time this worked.

**Comment by Novik** [comment]
@Micdu70 I already fixed this, thanks.

**Pull request merged**